index.js

```diff
@@ -17,7 +17,6 @@ const {
   toPullStream,
   toCallback,
 } = require('ssb-db2/operators')
-const pify = require('promisify-4loc')
```

Comment:
can this dep be removed as a dep?
any reason not to use core `util.promisify`

Reply:
Oh yeah, we could remove it now. The reason to use it over `require('utils').promisify` is that we need browser support. `promisify-4loc` is just 4 lines of code, as the name suggests, and supports browser and node.js (note that arj wants to keep ssb-browser-core working with the ssb stack, it's a great long-term goal to keep).